src/feature/coffeeChat/service/coffeeChatService.ts

```diff
@@ -33,25 +35,21 @@ export type CoffeeChatService = {
     token: string;
     coffeeChatContents: CoffeeChatRequest;
     postId: string;
-  }) => ServiceResponse<CoffeeChat>;
+  }) => ServiceResponse<CoffeeChatApplicant>;
   updateCoffeeChatStatus: ({
     token,
-    coffeeChatId,
     body,
   }: {
     token: string;
-    coffeeChatId: string;
-    body: { coffeeChatStatus: CoffeeChatStatus };
-  }) => ServiceResponse<CoffeeChat>;
+    body: { coffeeChatStatus: CoffeeChatStatus; coffeeChatList: string[] };
+  }) => ServiceResponse<CoffeeChatDetailList>;
   cancelCoffeeChat: ({
     token,
-    coffeeChatId,
     body,
   }: {
     token: string;
-    coffeeChatId: string;
-    body: { coffeeChatStatus: 'CANCELED' };
-  }) => ServiceResponse<CoffeeChat>;
+    body: { coffeeChatStatus: 'CANCELED'; coffeeChatList: string[] };
```

Comment:
cancelCoffeeChat이면 무조건 coffeeChatStatus가 'CANCELED'일텐데, 그럼 굳이 sevice 함수의 body에서 받을 필요가 있을까요? coffeeChatList만 받아도 괜찮을 것 같습니다.

```suggestion
   coffeeChatList: string[]
   // body 조합은 서비스 레이어에서 담당
```